azkaban-exec-server/src/main/java/azkaban/execapp/GuiceConstants.java

```diff
@@ -0,0 +1,8 @@
+package azkaban.execapp;
+
+public class GuiceConstants {
+
+  public static final String EXEC_JETTY_SERVER = "ExecServer";
```

Comment:
As long as it is a static variable, I don't think it should create a circular dependency since you don't need to instantiate the class to load a static constant. Could you give it a try and let me know. One of the injection tests should blow up in that case.